frontend/src/main.js

```diff
@@ -3,6 +3,7 @@
 import Vue from 'vue'
 import App from './App'
 import router from './router'
+import VuexRouterSync from 'vuex-router-sync'
```

Comment:
This is not enough. Look at [usage](https://github.com/vuejs/vuex-router-sync#usage).